packages/hash/frontend/src/pages/[account-slug]/types/entity-type/[entity-type-id].page.tsx

```diff
@@ -1,8 +1,14 @@
-import { extractBaseUri, extractVersion } from "@blockprotocol/type-system-web";
+import {
+  EntityType,
+  extractBaseUri,
+  extractVersion,
+} from "@blockprotocol/type-system-web";
 import { faAsterisk } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@hashintel/hash-design-system/fontawesome-icon";
 import { Box, Container, Typography } from "@mui/material";
+import { Buffer } from "buffer/";
```

Comment:
```suggestion
import { Buffer } from "buffer";
```

Not sure why this is not being linted 🤔

Reply:
Oh wow I did not know that! Interesting!